src/app/screens/login/index.tsx

```diff
@@ -22,12 +22,20 @@ const Logo = styled.img({
   height: 57,
 });
 
+const Button = styled.button({
+  background: 'none',
+});
+
 const ScreenContainer = styled(animated.div)({
   display: 'flex',
   flexDirection: 'column',
   flex: 1,
   paddingLeft: 18,
   paddingRight: 18,
+  overflowY: 'auto',
+  '&::-webkit-scrollbar': {
+    display: 'none',
+  },
```

Comment:
what was the reason for this?

Reply:
without this the default chrome scrollbar was being shown. Currently, files other than home already have this in their css, which is why no scrollbar gets shown on those screens. This was not required before since this CSS was also added in the `pages/Popup/index.css` which had to be removed in order to show the visible scrollbar 

https://github.com/secretkeylabs/xverse-web-extension/assets/88320460/4c4a9682-3f92-4faa-8ec4-986e6f16c907

Reply:
ah ok, got it